src/sql/finalize.sql

```diff
@@ -241,11 +349,24 @@ CREATE OPERATOR <=> (
 	COMMUTATOR = <=>
 );
 
+CREATE OPERATOR <=> (
+	PROCEDURE = _vectors_svecf32_operator_cosine,
+	LEFTARG = svector,
+	RIGHTARG = svector,
+	COMMUTATOR = <=>
+);
+
 -- List of functions
 
 CREATE FUNCTION pgvectors_upgrade() RETURNS void
 IMMUTABLE STRICT PARALLEL SAFE LANGUAGE c AS 'MODULE_PATHNAME', '_vectors_pgvectors_upgrade_wrapper';
 
+CREATE FUNCTION svector_from_split_array("dims" INT, "index" INT[], "value" real[]) RETURNS svector
+IMMUTABLE STRICT PARALLEL SAFE LANGUAGE c AS 'MODULE_PATHNAME', '_vectors_svector_from_split_array_wrapper';
+
+CREATE FUNCTION svector_from_kv_string("dims" INT, "input" TEXT) RETURNS svector
```

Comment:
```suggestion
CREATE FUNCTION svector_from_kv("dims" INT, "input" TEXT) RETURNS svector
```

Reply:
Remember to raise error if duplicate key found

Reply:
Already checked

https://github.com/tensorchord/pgvecto.rs/blob/65b83f0c2f3f20d252ce1af640d171266813e50e/src/datatype/svecf32.rs#L677-L687